kvdb/readwrite_bucket_test.go

```diff
@@ -151,15 +151,6 @@ func testBucketDeletion(t *testing.T, db walletdb.DB) {
 		// "apple/pear" deleted
 		require.Nil(t, apple.NestedReadWriteBucket([]byte("pear")))
 
-		// "apple/pear/cherry" deleted
```

Comment:
This is interesting. Isn't this actually a bbolt bug?

Reply:
Bolt crashes hard on it. I think it is just not pointing anywhere anymore.

Reply:
Hmm yes maybe. It isn't the most beautiful ending of a process.